primedev/server/serverpresence.h

```diff
@@ -45,10 +45,10 @@ struct ServerPresence
 class ServerPresenceReporter
 {
 public:
-	virtual void CreatePresence(const ServerPresence* pServerPresence) {}
-	virtual void ReportPresence(const ServerPresence* pServerPresence) {}
-	virtual void DestroyPresence(const ServerPresence* pServerPresence) {}
-	virtual void RunFrame(double flCurrentTime, const ServerPresence* pServerPresence) {}
+	virtual void CreatePresence(const ServerPresence* /*pServerPresence*/) {}
+	virtual void ReportPresence(const ServerPresence* /*pServerPresence*/) {}
+	virtual void DestroyPresence(const ServerPresence* /*pServerPresence*/) {}
+	virtual void RunFrame(double /*flCurrentTime*/, const ServerPresence* /*pServerPresence*/) {}
```

Comment:
Huh?

Reply:
I figured it would be dumb to have a function body of literally only `NOTE_UNUSED`. honestly these should probably be pure virtual?